tests/unit/TodoList1.spec.ts

```diff
@@ -255,6 +255,7 @@ describe('TodoList - UI', () => {
     const wrapper = mount(TodoList, { vuetify, mixins: [AllowableActionsMixin] })
     const vm = wrapper.vm as any
     await Vue.nextTick()
+    await flushPromises()
```

Comment:
Is this required because of the multiple asyncs in loadData()?

Theoretically this additional wait shouldn't be needed, but if you can't figure this out quickly then I recommend leaving it and remove the await nextTick() calls (you really shouldn't need both; if you do then something's wrong with the code or test).

Reply:
Yes, it is required because of the await in loadData. 

I believe we could do without the nextTick() but not without flushPromises now.

What I understand is that nextTick waits for next updates of the DOM for vue (which can happen multiple times) while flushPromises waits for clearing of all the promises currently in queue (including the DOM update)

OK will remove nexTick() calls